src/core/ext/filters/channel_idle/legacy_channel_idle_filter.cc

```diff
@@ -296,13 +296,13 @@ const grpc_channel_filter LegacyMaxAgeFilter::kFilter =
 
 void RegisterLegacyChannelIdleFilters(CoreConfiguration::Builder* builder) {
   builder->channel_init()
-      ->RegisterFilter<LegacyClientIdleFilter>(GRPC_CLIENT_CHANNEL)
+      ->RegisterV2Filter<LegacyClientIdleFilter>(GRPC_CLIENT_CHANNEL)
       .ExcludeFromMinimalStack()
       .If([](const ChannelArgs& channel_args) {
         return GetClientIdleTimeout(channel_args) != Duration::Infinity();
       });
   builder->channel_init()
-      ->RegisterFilter<LegacyMaxAgeFilter>(GRPC_SERVER_CHANNEL)
+      ->RegisterV2Filter<LegacyMaxAgeFilter>(GRPC_SERVER_CHANNEL)
```

Comment:
Aren't we still going to want the max-age filter in v3?  This one is server-side, not client-side, so it's not changing due to the new channel architecture, is it?

Reply:
I think there's some good reasons to move this up into server (a little different to client, but similar): for example, we can probably rely on a single periodic timer to bulk check all connections for max age, and reduce constant background timer pressure in the system.